bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:broadlink:config">
+		<parameter name="ipAddress" type="text">
+			<context>network-address</context>
+			<label>Network Address</label>
+			<description>Network address of the device.</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="staticIp" type="boolean">
+			<context>static-ip</context>
+			<label>Static IP</label>
+			<description>Will the device always be given this network address?</description>
```

Comment:
Interesting config to have, can you provide some background why this is used or needed? Usually I just make sure that the address config supports both IP and also hostnames, so a hostname can be used if it is not static.

Reply:
I have no idea why this is in here. Since parts of this code is already 5+ years old, I am a bit hesitant to remove functionality, as people might want to use it.